pkg/client/model_customer.go

```diff
@@ -28,13 +28,32 @@ type Customer struct {
 	// Customers name suffix. \"Jr\", \"PH.D.\"
 	Suffix string       `json:"suffix,omitempty"`
 	Type   CustomerType `json:"type"`
+	// Business Name for business type customers
```

Comment:
I'm not sure if this is in the scope of the PR, but:
We should consider composing fields related to the business profile into its own struct. And you can shorten certain field names too.
```go
type Business struct {
 CustomerID string
 Name string // was BusinessName
 Type string // was BusinessType
... 
}
```

Reply:
I think this would only apply to Businesses. Also, it'd be a breaking change and would likely change our JSON responses to api.moov.io

Reply:
I thought about using a `OneOf` approach, which is available in the OpenAPI docs.  We could do one of business or individual and and `AllOf` for both of those types would have the shared properties of customer, or something like that

https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

Reply:
But it would be difficult to make that backwards compatible.  Do we plan to ever version our endpoints @adamdecaf ?

Reply:
We do version api.moov.io endpoints, but none of our apps have needed a versioned endpoint.